gnd/src/main/res/values/strings-untranslated.xml

```diff
@@ -21,5 +21,6 @@
   <string name="app_name" translatable="false">Ground</string>
   <string name="text_separator" translatable="false">·</string>
   <string name="ground_website" translatable="false">https://groundplatform.org/</string>
+  <string name="ground_android_github" translatable="false">https://github.com/google/ground-android/issues/new/choose</string>
```

Comment:
Maybe we should default to **bug report**? I suppose users could also want to request a feature, but they definitely won't report code health issues. I think reporting bugs is the most likely use case.

Reply:
@gino-m Please share your thoughts too. My rationale behind this was to promote users to suggest features in addition to reporting bugs.